diff --git a/setup.py b/setup.py
index c2d65897..794fb4fa 100644
--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,7 @@ requirements = [
     "setuptools>=41.0.0",  # to satisfy dependency constraints
     "tabulate",
     "tensorflow-probability[tf]>=0.12.0",
-    "tensorflow>=2.4.0; platform_system!='Darwin' or platform_machine!='arm64'",
+    "intel-tensorflow>=2.4.0; platform_system!='Darwin' or platform_machine!='arm64'",
     # NOTE: Support of Apple Silicon MacOS platforms is in an experimental mode
     "tensorflow-macos>=2.4.0; platform_system=='Darwin' and platform_machine=='arm64'",
     # NOTE: once we require tensorflow-probability>=0.12, we can remove our custom deepcopy handling
